compiler/test/runnable_cxx/test7925.d

```diff
@@ -113,11 +113,11 @@ void main()
     assert(dg3(20, 3) == 123);
     assert(dg4(20, 3, 0) == 123);
 
-    assert(callMember!(C1.f0)(c1) == 100);
-    assert(callMember!(C1.f1)(c1, 1) == 101);
-    assert(callMember!(C1.f2)(c1, 20, 3) == 123);
-    assert(callMember!(C1.f3)(c1, 20, 3) == 123);
-    assert(callMember!(C1.f4)(c1, 20, 3, 0) == 123);
+    //assert(callMember!(c1.f0)(c1) == 100);
```

Comment:
This seems to test the same thing as the above delegate test. What am I missing @ibuclaw ?